.github/workflows/ensure-triage-label.yml

```diff
@@ -0,0 +1,30 @@
+name: Ensure triage label is present
+on:
```

Comment:
🤔  I'm just realizing that a small improvement we perhaps could do is to trigger it also on label deletion https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#label. To make sure an issue always has at least this label

```yml
  label:
    types:
      - deleted
```

Reply:
Added the trigger